### DISCUSSION TITLE
How to switch to dark mode?

### DISCUSSION THREAD

**Discussion by jkibort928** [PintaProject/Pinta#735] [Q&A] [answered]
I can't seem to find a settings/preferences tab for stuff like this, is there a dark mode option?

**Comment by cameronwhite** [comment, marked as answer]
In Pinta 2.1 there is not a setting for this, so Pinta's appearance depends on your system's GTK theme which might not be easy to adjust depending on your platform.
In the Pinta 2.2 development builds (which use GTK4 / libadwaita) there is now a configuration option for a light vs dark theme (#613)

The latest dev builds can be downloaded from https://github.com/PintaProject/Pinta/actions?query=branch%3Amaster

**Comment by Novack** [comment]
Thank you! Finding this was harder than it seems.
